### DISCUSSION TITLE
Where can find original email templates and overwrite them

### DISCUSSION THREAD

**Discussion by newathnow** [directus/directus#17526] [Archive]
### Describe the Bug

I am trying to find the original email templates but cant find them. 

Want to change the style and add new one based on documentation:
https://docs.directus.io/extensions/email-templates.html

but i dont know what they have inside as tags 

### To Reproduce

its just question

### Errors Shown

its just question

### What version of Directus are you using?

9.22.3

### What version of Node.js are you using?

&16

### What database are you using?

sqlite

### What browser are you using?

brave

### How are you deploying Directus?

locally

**Comment by newathnow** [comment]
hey team :D at least something in documentation or some guide will be useful 😅😅

**Comment by jtlindsey** [comment]
The only `.liquid` files i could find are in `api/src/services/mail/templates` of the [original source code](https://github.com/directus/directus/tree/main/api/src/services/mail/templates).

<img width="226" alt="image" src="https://user-images.githubusercontent.com/7409277/219877557-10a5d1b3-ecd8-460f-aadc-fc6457fe871a.png">

I haven't had time to try it yet but i imagine copying one of these to your `/extensions/templates/<template-name>.liquid` would override the original.

**Reply by newathnow** [reply]
thank you so much mate appreciate it came just in time :D for some reason when i searched *.liquid in ubuntu and on my mac doesnt appeared. save my day :) owe u beer

**Reply by teichsta** [reply]
have you been successful with overriding a template? I copied (and translated) a template into a new templates folder but the translated liquid-file doesn't seem to get pulled properly. Is there any special configuration needed @jtlindsey ?

**Reply by jtlindsey** [reply]
> have you been successful with overriding a template? I copied (and translated) a template into a new templates folder but the translated liquid-file doesn't seem to get pulled properly. Is there any special configuration needed @jtlindsey ?

I have not tried to override a template.

**Reply by teichsta** [reply]
and you @newathnow does it work for you?

**Reply by jtlindsey** [reply]
If you have a minimal reproduction on github i can look at i might be able to help.

**Reply by newathnow** [reply]
> and you @newathnow does it work for you?

 just replaced the files with same names mate but if u want another types of emails must do flow or in my case i used hooks that call api endpoint and fire email based on your needs

**Reply by Gustavo-Kuze** [reply]
For anyone having trouble with overwriting the files using Docker container:

- Create new files based on the [original ones](https://github.com/directus/directus/tree/main/api/src/services/mail/templates) in a local directory
- Modify them as needed
- Find out your running docker container ID with command `docker ps -a`
- Copy the files to the  running container:

```
docker cp base.liquid {DOCKER_CONTAINER_ID}:/directus/dist/services/mail/templates/base.liquid
docker cp password-reset.liquid {DOCKER_CONTAINER_ID}:/directus/dist/services/mail/templates/password-reset.liquid
```

**NOTE**: Remember to replace `{DOCKER_CONTAINER_ID}` with the ID of your running container.

**Comment by alexis-karyavin** [comment]
I managed to replace the basic template. I use docker. I created it based on this [guide](https://docs.directus.io/self-hosted/docker-guide.html)

To begin with, I redefined the env variable so that directus would understand where to get templates from.

`EMAIL_TEMPLATES_PATH: "/directus/extensions/templates"`

Next, I created a folder where I put all my templates.

<img width="345" alt="изображение" src="https://github.com/user-attachments/assets/0d69a5b1-6529-47b3-8ac7-43d8551d9921" />

My base.liquid looks like this lifestyle

<img width="356" alt="изображение" src="https://github.com/user-attachments/assets/038034da-0974-4f8d-9304-f03d8cdf75c2" />

Next, I connect the layout as follows

{% layout "custom-name/base" %}

{% block content %}
  Your content
{% endblock %}`

Result:
<img width="346" alt="изображение" src="https://github.com/user-attachments/assets/bca21663-f8b2-4006-ab12-594739519d45" />
